src/taco-cli/test/kit.ts

```diff
@@ -64,6 +73,22 @@ describe("Kit", function (): void {
     var tempJson: string = path.resolve(runFolder, "temp.json");
     var originalCwd: string;
 
+    var expectedCliTacoJsonKeyValues1: IKeyValuePair<string> = {
+        "cordova-cli": "4.3.1"
+    };
+
+    var expectedCliTacoJsonKeyValues2: IKeyValuePair<string> = {
+        "cordova-cli": "5.1.1"
+    };
+
+    var expectedKitTacoJsonKeyValues: IKeyValuePair<string> = {
+        kit: "5.1.1-Kit", "cordova-cli": "5.1.1"
+    };
+
+    var expectedKitPlatformVersion: IComponentVersionMap = {
```

Comment:
Why does this have a different type to the others? looks like it is actually equivalent, is that right?